src/server/ApolloServer.ts

```diff
@@ -52,11 +58,32 @@ export class ApolloServer {
     this.server.use(bodyParser());
 
     const schema = await buildSchema({
-      resolvers: [ItemResolver],
+      globalMiddlewares: [DataLoaderMiddleware],
+      resolvers: [
+        path.resolve(__dirname, '../resolvers/**/resolver.ts'),
+      ],
+      authChecker,
       dateScalarMode: 'timestamp',
     });
 
-    const apolloServer = new ApolloServerKoa({ schema });
+    const apolloServer = new ApolloServerKoa({
+      schema,
+      tracing: true,
```

Comment:
Should we commit `tracing: true` into git repo or just debug in local?

Reply:
just debug!